src/calctree/calculation_result.py

```diff
@@ -0,0 +1,54 @@
+import json
+from typing import List
+
+from src.calctree.exceptions import ParameterNotFoundException
+
+
+class CalculationResult:
```

Comment:
Most of these docstrings are just describing types - why not use the type hinting to do it?